package ssp;
message RelationInfo {
	optional int32 verion = 1;
	optional int32 user_id = 2;
	repeated int32 friend_user_id = 3;
	repeated int32 black_user_id = 4;
}